src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/Orchestrator/EnvironmentVariableAnalyzer.cs

```diff
@@ -20,44 +21,37 @@ public class EnvironmentVariableAnalyzer: DiagnosticAnalyzer
         private const string Category = SupportedCategories.Orchestrator;
         public const DiagnosticSeverity severity = DiagnosticSeverity.Warning;
```

Comment:
General C# convention is to make constant variable names capitalized - i.e. `Severity` instead of `severity`. Same thing for this file and other files.